### PR TITLE
Feat(InvestigationTable): Phone displays as icon and tooltip

### PR DESCRIPTION
**Pull request by doronlnd84**
Modified commentDisplay (now - imageTooltipDisplay) to be generic and use it both for Comment and Phone

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 1]
```diff
@@ -344,8 +344,18 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                 <div>{subOccupation || '-'}</div>
                             </Tooltip>    
             case TableHeadersNames.comment:
-                return <CommentDisplay comment={indexedRow[cellName as keyof typeof TableHeadersNames]}
-                    scrollableRef={tableContainerRef.current} />
+                return <ImageTooltipDisplay value={indexedRow[cellName as keyof typeof TableHeadersNames]}
+                    defaultValue={'אין הערה'}
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:348]: dont need the curly braces

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 2]
```diff
@@ -344,8 +344,18 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
+                    scrollableRef={tableContainerRef.current}
+                    InputIcon={Comment}
+                     />
+            case TableHeadersNames.phoneNumber:
+                        return <ImageTooltipDisplay 
+                                value={indexedRow[cellName as keyof typeof TableHeadersNames]}
+                                defaultValue={''}
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:355]: same

File: client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import {Comment} from '@material-ui/icons';
-import {Tooltip, ClickAwayListener, IconButton} from '@material-ui/core';
+//  import {Comment} from '@material-ui/icons';
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx:2]: delete comment

File: client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+import {Tooltip, ClickAwayListener, IconButton, SvgIconTypeMap} from '@material-ui/core';
 
-import useStyles from './commentDisplayStyles';
+import useStyles from './imageTootipDisplayStyles';
+import { OverridableComponent } from '@material-ui/core/OverridableComponent';
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx:6]: imports order 
add it to line 3

**Review by ItayBenMoshe — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx [hunk 5]
```diff
@@ -1,10 +1,8 @@
 import React from 'react';
-//  import {Comment} from '@material-ui/icons';
+import { OverridableComponent } from '@material-ui/core/OverridableComponent';
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx:2]: You can unite between the imports
`import {Tooltip, ClickAwayListener, IconButton, SvgIconTypeMap, OverridableComponent} from '@material-ui/core';`

**Review by ItayBenMoshe — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx (hunk 5 above)
Comment (doronlnd84) [reply client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx:2]: This is actually causes compile error, try it.

**Review by ItayBenMoshe — APPROVED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 6]
```diff
@@ -344,8 +344,18 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                 <div>{subOccupation || '-'}</div>
                             </Tooltip>    
             case TableHeadersNames.comment:
-                return <CommentDisplay comment={indexedRow[cellName as keyof typeof TableHeadersNames]}
-                    scrollableRef={tableContainerRef.current} />
+                return <ImageTooltipDisplay value={indexedRow[cellName as keyof typeof TableHeadersNames]}
+                    defaultValue='אין הערה'
+                    scrollableRef={tableContainerRef.current}
+                    InputIcon={Comment}
+                     />
```
Comment (RoeiRom) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:351]: Indentation

File: client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx [hunk 7]
```diff
@@ -38,18 +36,20 @@ const CommentDisplay = ({comment, scrollableRef}: Props) => {
...
       </ClickAwayListener>
     )
 }
 
 interface Props {
-    comment: string | null;
+    value: string | null;
+    defaultValue?: string;
     scrollableRef?: HTMLElement;
+    InputIcon: OverridableComponent<SvgIconTypeMap<{}, "svg">>;
```
Comment (RoeiRom) [inline client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx:52]: why OverrideableComponent and not just an svg icon component?

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 8]
```diff
@@ -344,8 +344,18 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
+            case TableHeadersNames.phoneNumber:
+                        return <ImageTooltipDisplay 
+                                value={indexedRow[cellName as keyof typeof TableHeadersNames]}
+                                defaultValue=''
+                                scrollableRef={tableContainerRef.current}
+                                InputIcon={Call}
+                             />
```
Comment (RoeiRom) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:358]: Why not using the phone component?

Comment (RoeiRom) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:358]: It is a common component

**Review by RoeiRom — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx [hunk 9]
```diff
@@ -1,12 +1,10 @@
...
+import { OverridableComponent } from '@material-ui/core/OverridableComponent';
+import {Tooltip, ClickAwayListener, IconButton, SvgIconTypeMap} from '@material-ui/core';
 
-import useStyles from './commentDisplayStyles';
+import useStyles from './imageTootipDisplayStyles';
 
-const noCommentMessage = 'אין הערה';
-
-const CommentDisplay = ({comment, scrollableRef}: Props) => {
+const CommentDisplay = ({value, defaultValue, scrollableRef, InputIcon}: Props) => {
```
Comment (sandymash) [inline client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx:7]: if you change file name also change component name

Comment (sandymash) [reply client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx:7]: also, imageTooltipDisplay isnt very clear... maybe icon with tooltip?

Comment (sandymash) [reply client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx:7]: or clickable tooltip

**Comment by doronlnd84** [comment]
there is  new one

**Pull request closed without merging**